app/experimenter/experiments/email/legacy.py

```diff
@@ -90,6 +90,20 @@ def send_experiment_comment_email(comment):
     )
 
 
+def send_experiment_change_email(change):
+
+    # Launch has its own separate email
+    if str(change) not in ("Launched Experiment"):
```

Comment:
If you wanna do tuple membership check, you need a comma here:

`("Launched Experiment")` evaluates as an expression, not a tuple: `"LaunchedExperiment"` so it's checking that `str(change)` string is a substring of `"LaunchedExperiment"`, which might also work but I don't think is exactly what you meant by the way it's written.

If you wanna do membership you gotta do

`if str(change) not in ("Launched Experiment",):`

It's kinda dumb in Python that `(a)` is an expression and the parens just go away vs `(a,)` which is a tuple of length 1.

Reply:
oh no, this was not what I wanted 🤦‍♀️ results of tinkering... will fix